src/components/Todo/TodoViews/todo-views.jsx

```diff
@@ -1,14 +1,28 @@
 import React from "react";
 import PropTypes from "prop-types";
 import IncompleteTask from "components/Todo/TodoViews/incomplete-task.component";
+import CompletedTask from "components/Todo/TodoViews/completed-task.component";
 import TodoDetails from "components/Todo/TodoViews/todo-details.component";
 
-const TodoViews = ({ todoList, handleDeleteTask }) => {
+const TodoViews = ({ todoList, handleDeleteTask, handleCompleteTask }) => {
   return todoList.map((list) => {
```

Comment:
While traversing over the `todoList` what get each time can be denoted as a **todo** or an **item**.

```suggestion
  return todoList.map((todo) => {
```